tox.ini

```diff
@@ -1,15 +1,15 @@
 # Copyright (C) 2019, Nokia
 
 [tox]
-envlist = py27, py36, docs, pylint
```

Comment:
Should not remove `py36` as it seems to be still supported by the tool and we even provide `run_in_virtualenv3.6`.
On the other hand, adding `py37` is fine, we could even add it to `get_python_versions`.

Reply:
OK